op-service/dial/active_l2_provider_test.go

```diff
@@ -0,0 +1,104 @@
+package dial
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+// TestActiveSequencerFailoverBehavior_RollupProvider tests that the ActiveL2RollupProvider
+// will failover to the next provider if the current one is not active.
+func TestActiveSequencerFailoverBehavior_RollupProviders(t *testing.T) {
```

Comment:
A second test would be nice where both sequencers are unreachable like 2 times and then only on the third attempt one is online.

However, maybe we also decide to simplify the impl to not run multiple attempts over the list at all.

Reply:
Resolved: new design (for now) only checks the whole list once.